lib/Mojolicious/Guides/Cookbook.pod

```diff
@@ -178,19 +178,29 @@ worker is allowed to handle (often as low as C<1>).
   $ ./script/my_app prefork -m production -w 10 -c 1
   Web application available at http://127.0.0.1:3000
 
-During startup your application is preloaded in the manager process, which does not run an event loop, so you can use
-L<Mojo::IOLoop/"next_tick"> to run code whenever a new worker process has been forked and its event loop gets started.
+During startup your application is preloaded in the manager process
+but it will not start any processing there.
+
+You can use
+the hook L<Mojolicious/"before_worker_start"> to run code in each child
+process (the worker processes that are being forked) before it is about
+to start processing incoming requests:
```

Comment:
Reformatting.

Reply:
Done, reformatted to 80 columns.

Reply:
We use 120 columns.

Reply:
Reformatted to 120 columns since it is the standard for this project.